Add section 1.1.2 (#9)
jneidel/internet.org-entwicklung: main.tex

% 1.1
\subsection{Internet.org/Free Basics}

% 1.1.2
\subsubsection{Konzept}
        
Mark Zuckerberg nennt drei Punkte, an denen man arbeiten muss, damit sich das Internet in den Entwicklungsländern ausbreiten kann
\parencite{HumanRight}:

\begin{itemize}
\item ``making internet access affordable by making it more efficient to deliver data."
\item ``using less data by improving the efficiency of the apps and experiences we use."
\item ``helping businesses drive internet access by developing a new model to get people online."  
\end{itemize}

> Die ersten zwei Punkte, die er erwähnt, zielen darauf ab das Internet kostengünstiger/kostenlos anzubieten. 

Reviewer: ```suggestion
Der erste Punkt zielt darauf ab, das Internet kostengünstiger/kostenlos anzubieten. 
```

> Dies ist notwendig damit Menschen, die nur ein geringes Einkommen haben, es auch nutzen können.

Reviewer: ```suggestion
Dies ist notwendig, damit auch Menschen mit einem geringen Einkommen es nutzen können.
```

Dazu ist es zunächst nötig den Datenverkehr effizienter gestalten, zum Beispiel durch leistungsfähigere Hardware.

> Dies senkt Infrastruktur Kosten für die Unternehmen die diese bereitstellen.

Reviewer: ```suggestion
Dies senkt Infrastrukturkosten für die Unternehmen, die diese bereitstellen.
```

Wenn dieser Profit an die Kunden weitergereicht wird führt das zu einer Senkung der Internetkosten für den Einzelnen.

> Zusätzlich könnten die überschüssigen Profit zum weiteren Ausbau des Netzwerkes genutzt werden.

Reviewer: ```suggestion
Zusätzlich könnte das dadurch entstehende überschüssige Kaptial in den weiteren Ausbau des Netzwerkes investiert werden.
```

\medskip

> Im zweite Punkt konzentriert sich Zuckerberg auf die erhöhung der Leistungsfähigkeit von existierenden Dienstleistungen.

Reviewer: ```suggestion
Im zweite Punkt konzentriert sich Zuckerberg auf die Erhöhung der Leistungsfähigkeit von existierenden Dienstleistungen.
```

> Es ist logisch: weniger übertragene Daten entsprechen geringeren Kosten für den Nutzer.

Reviewer: ```suggestion
Es ist logisch: Weniger übertragene Daten entsprechen geringeren Kosten für den Nutzer.
```
Vollständiger Satz nach einem Doppelpunkt: Großschreibung (I dunno either)

\begin{quote}
``At Facebook, we’re investing heavily in opportunities to reduce our overall data use and help  other apps     
reduce their data use as well. Some of the areas we’re focused on are caching, data compression and simple efficiency optimizations.
- Mark Zuckerberg \textcite[8]{HumanRight}
\end{quote}

% cut

> Die angesprochene Komprimierung kann man entweder durch die direkte Kompression von großen Apps realisieren oder man könnte neue Dienstleistungen entwickeln, durch welche man seine Daten schicken und Komprimieren lassen kann.

Reviewer: ```suggestion
Die angesprochene Komprimierung kann man entweder durch die direkte Kompression von großen Apps realisieren oder indem man neue Dienstleistungen entwickelt, durch welche man seine Daten schicken und komprimieren lassen kann.
```

Facebook hat diese Mittel verwendet um den Datenverkehr effizienter zu gestalten.

> Die Projekte Facebook Zero (Bereitstellung einer kostenlosen Internetverbindung solange man auf Facebooks Webseiten ist) und ``Facebook für jedes Handy" (eine Version von Facebook für feature Phones) wurden 2010 und 2011 eingeführt.

Reviewer: ```suggestion
Die Projekte Facebook Zero (Bereitstellung einer kostenlosen Internetverbindung, solange man sich auf Facebooks Webseiten befindet) und ``Facebook für jedes Handy" (eine Version von Facebook für feature Phones) wurden jeweils  2010 und 2011 eingeführt.
```

> Die verwendete Technologie will Facebook nun allgemein anwendbar machen, damit andere Dienstleistungen diese auch benutzen können.

Reviewer: ```suggestion
Facebook will die verwendete Technologie nun allgemein anwendbar machen, damit andere Dienstleistungen diese auch benutzen können.
```

\medskip

> Mit dem letzen Punkt sagt Facebook, dass Sie glauben das die Ausbreitung des Internets die Aufgabe der Industrie ist, denen Facebook mit Konzepten wie Internet.org zu Seite stehen möchte.

Reviewer: ```suggestion
Mit dem letzten Punkt bekräftigt Zuckerberg den Glauben, die Ausbreitung des Internets sei die Aufgabe der Industrie , der Facebook mit Konzepten wie Internet.org zu Seite stehen möchte.
```

> Dieses neue Konzept wurde in wie fern umgesetzt: Menschen in Entwicklungsländern sollen zuerst nur den kostenlosen Zugang zu grundlegenden Dienstleistungen wie Facebook haben.

Reviewer: ```suggestion
Dieses neue Konzept wurde dermaßen umgesetzt: Menschen in Entwicklungsländern sollen zuerst nur den kostenlosen Zugang zu grundlegenden Dienstleistungen wie Facebook haben.
```

> Für diese Herangehensweise wurden zwei Begründungen gegeben:

Reviewer: ```suggestion
Für diese Herangehensweise gab Facebook zwei Begründungen:
```

\medskip

> Nicht jegliche Dienstleistung kann kostenlos bereitgestellt werden, da die dahinterstehende Anbieter Profit erzielen müssen. 

Reviewer: ```suggestion
Nicht jede Dienstleistung kann kostenlos bereitgestellt werden, da die dahinterstehenden Anbieter Profit erzielen müssen. 
```

> Aber durch das kostenlose Bereitstellen grundlegender Anwendungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Zugang zum Internet bieten kann).

Reviewer: ```suggestion
Aber durch das kostenlose Bereitstellen grundlegender Anwendungen sei es möglich, den meisten Menschen Zugang zum Internet und der Industrie trotzdem indes den dabei größtmöglichen wirtschaftlichen Profit zu gewährleisten (wodurch diese wiederum mehr Zugang zum Internet bieten können).
```

> Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.

Reviewer: ```suggestion
Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internetdienstleistungen vorgibt, beschreibt er, welche Anwendungen er als grundlegend ansieht.

\begin{quote}
``many people who have never experienced the internet don’t know what a data plan is or why they’d want one. 
However, most people have heard of services like Facebook and messaging and they want access to them. If we can provide people 
with access to these services, then they’ll discover other content they want and begin to use 
and understand the broader internet." - \textcite[4]{HumanRight}
\end{quote}

```

> Seiner Meinung nach sind es die Dienstleistungen, die nicht viel Daten verbrauchen (hauptsächlich Text basierende und einfach gehaltene Apps) und durch welche anderer Inhalt entdeckt werden kann.

Reviewer: ```suggestion
Seiner Meinung nach sind es solche Dienstleistungen, die nicht viel Datenvolumen verbrauchen (hauptsächlich textbasierte und/oder einfach gehaltene Apps) und gleichzeitig auf andere Inhalte des Internets hin- oder verweisen.
```

> Dienstleistungen die unter diesen Kriterien fallen sind zum Beispiel Messengers, soziale Netzwerke, Suchmaschinen und Wikipedia.(\cite[5]{HumanRight})

Reviewer: ```suggestion
Dienstleistungen, die unter diese Kriterien fallen, sind zum Beispiel Instant-Messaging-Dienste und soziale Netzwerke, Suchmaschinen und Enzyklopädien wie Wikipedia.(\cite[5]{HumanRight})
```

> Also ist Facebook nach Zuckerbergs Meinung eine grundlegende Internet Dienstleistung, die vorrangig zu anderen Anwendungen kostenlos zur Verfügung gestellt werden sollte.

Reviewer: ```suggestion
Dementsprechend ist Facebook Zuckerberg nach ein grundlegender Service, der eher als Andere kostenlos und überall zur Verfügung gestellt werden sollte.
```

Aber durch das kostenlose Bereitstellen grundlegender Anwendungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Menschen den Zugang zum Internet ermögliche können).
Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.

\begin{quote}
``many people who have never experienced the internet don’t know what a data plan is or why they’d want one. 
However, most people have heard of services like Facebook and messaging and they want access to them. If we can provide people 
with access to these services, then they’ll discover other content they want and begin to use 
and understand the broader internet." - \textcite[4]{HumanRight}
\end{quote}

> Facebooks Plan ist es diese Punkte unter dem Projekt Internet.org Industrie übergreifend zu bearbeiten.

Reviewer: Delet this.

Author: was genau?

Reviewer 2: I finde das gut, rundet das Zitat ab

Reviewer: @SuFischer  99-108 are dublicate 
At least in part

% 1.1
\subsection{Internet.org/Free Basics}

% 1.1.2

> \subsubsection{Geplante Umsetzung}

Reviewer 2: ```suggestion
\subsubsection{Konzept}
```

Hört sich griffiger an

        
Mark Zuckerberg nennt drei Punkte, an denen man arbeiten muss, damit sich das Internet in den Entwicklungsländern ausbreiten kann

> (\cite{HumanRight})    

Reviewer 2: ```suggestion
\parencite{HumanRight}:
```

\begin{itemize}
\item ``making internet access affordable by making it more efficient to deliver data."

\item ``using less data by improving the efficiency of the apps and experiences we use."

\item ``helping businesses drive internet access by developing a new model to get people online."  
\end{itemize}

Die ersten zwei Punkte, die er erwähnt, zielen darauf ab das Internet kostengünstiger/kostenlos anzubieten. 

> Dies ist notwendig damit Menschen, die nur ein minimales Einkommen haben, es auch benutzen können.

Reviewer 2: ```suggestion
Dies ist notwendig damit Menschen, die nur ein geriges Einkommen haben, es auch nutzen können.
```

> %Formatierung

Reviewer 2: ```suggestion
```

\medskip

> Zunächst müsste man den Datenverkehr effizienter gestalten, zum Beispiel durch leistungsfähigere Hardware.

Reviewer 2: Was heißt das konkret? Ich kann mir nichts darunter vorstellen.

Dadurch würden sich die Kosten für die Infrastruktur verringern und die Unternehmen, die diese bereitstellen, würden mehr Profit    

> machen. Dies würde zu einer Senkung der Internet Kosten für den einzelnen führen.

Reviewer 2: ```suggestion
machen. Dies würde zu einer Senkung der Internetkosten für den Einzelnen führen.
```

> Zusätzlich könnten die Unternehmen den überschüssigen Profit zum weiteren Ausbau des Netzwerkes benutzen\\

Reviewer 2: Hahaha, Freiwilligkeit kappt immer super! Siehe die Deutsche Bahn ;)

> Facebook selber möchte auch an dem Ausbau der Internet Infrastruktur arbeiten ( mithilfe von Drohnen und Satelliten - die obtische    

Reviewer 2: ```suggestion
Facebook selber möchte auch an dem Ausbau der Internet Infrastruktur arbeiten (mithilfe von Drohnen und Satelliten, welche optische    
```

> Freiraumkommunikation benutzen), aber das Primärziel ist es Menschen die bereits in Reichweite von Internetzugängen leben, zu

Reviewer 2: `in Reichweite von Internetzugängen leben` klingt komisch und ergibt auch wenig Sinn, denn Funkturm + SIM Karte würde ich nicht als Internetzugang bezeichnen, zumindest nicht als einer dem man physisch nah sein kann.

verbinden.\\
Also wollen sie sich darauf konzentrieren die Kosten für die Telekommunikationsfirmen zu senken.
Sie sind bereits Mitwirkende an dem Open Compute Project, welches ein Projekt ist um Leistungsfähige und kostengünstige Hardware 

> Design mit unterschiedlichen Firmen zu teilen. \\

Reviewer 2: Hier fehlen die Quellen.

Reviewer 2: `Comment on lines x to y` über dem Kommentar beachten

> Facebook arbeitet außerdem daran web-caching(Zwischenspeicherung der benötigten Daten in den Datenzentren) den grundlegenden 

Reviewer 2: ```suggestion
Facebook arbeitet außerdem daran Web Caching (Zwischenspeicherung der benötigten Daten in den Datenzentren) der grundlegenden 
```

> Internetdienstleistungen anzubieten, dadurch wird Zeit gespart und der Datenverkehr verringert sich.\\

Reviewer 2: Was sind denn die `grundlegenden Internetdienstleistungen`, das klingt komisch :)

> Facebook nimmt an, dass mit einem organisierten Einsatz es möglich sei, die Leistungsfähigkeit von dem Zustellen von Daten in den   

Reviewer 2: ```suggestion
Facebook nimmt an, dass es mit einem organisierten Einsatz möglich sei, die Leistungsfähigkeit von dem Zustellen von Daten in den
```

> nächsten 5 bis 10 Jahren um das 100 fache zu erhöhen  (vgl. \cite{HumanRight})\\

Reviewer 2: `die Leistungsfähigkeit von dem Zustellen` I don't understand...

Unverständlich und sehr englisch :smile:

Reviewer 2: Kann vlt. auch weggelassen werden.

\medskip

> Der zweite Punkt auf den Zuckerberg eingeht konzentriert sich auf die Leistungsfähigkeit der angebotenen Dienstleistungen und wie man    diese verbessern kann. Es ist logisch: wenn man weniger Daten für die Internetnutzung braucht ist sie kostengünstiger.\\

Reviewer 2: ```suggestion
Der zweite Punkt auf den Zuckerberg eingeht konzentriert sich auf die Leistungsfähigkeit der angebotenen Dienstleistungen und wie man diese verbessern kann. Es ist logisch: wenn man weniger Daten für die Internetnutzung braucht ist sie kostengünstiger.\\
```

Reviewer 2: `Es ist logisch: wenn man weniger Daten für die Internetnutzung braucht ist sie kostengünstiger.`

Das klingt 'kopiert-übersetzt', bitte umformulieren, z.b.:
`Es ist logisch: weniger Daten zu Übertragen entspricht weniger verursachten Kosten.`

> Dienstleistungen kann man auf verschiedenen Weisen effizienter gestalten\\

Reviewer 2: Wieso steht der Satz hier allein auf einsamer Flur?
Wenns zum Obigen Absatz gehört, dort integrieren, sonst flieg der raus :)

> Mark Zuckerberg sagt ``At Facebook, we’re investing heavily in opportunities to reduce our overall data use and help  other apps     

Reviewer 2: ```suggestion
Mark Zuckerberg sagt:

``At Facebook, we’re investing heavily in opportunities to reduce our overall data use and help  other apps     
```

> reduce their data use as well. Some of the areas we’re focused on are caching, data compression and simple efficiency optimizations.

Reviewer 2: ```suggestion
reduce their data use as well. Some of the areas we’re focused on are caching, data compression and simple efficiency optimizations. - \textcite{HumanRight}
```

> (\cite{HumanRight})\\

Reviewer 2: ```suggestion

```

Die Daten Komprimierung kann man entweder durch die direkte Kompression von großen Apps realisieren oder man könnte neue Dienstleistungen entwickeln, durch welche man seine Daten schicken und Komprimieren lassen kann.\\

\medskip

> Mit dem dritten und letzten Punkt hat Zuckerberg eine spezifische Sache umschrieben.

Reviewer 2: ```suggestion
Mit dem dritten und letzten Punkt hat Zuckerberg eine genauere Idee umschrieben.
```

Das ``new model" von dem er spricht kann man in einem Satz zusammenfassen:

> Die Menschen, in den Entwicklungsländern sollen erst mal nur den kostenlosen Zugang zu Dienstleistungen wie Facebook haben.\\

Reviewer 2: Habe ich anders verstanden.

Für geht es bei `helping businesses drive internet access by developing a new model to get people online.`
darum, das Firmen den Internetzugang bereit stellen, indem sie sich neue Konzepte ausdenken um die Leute mitzunehmen. Das hat erstmal nichts mit FB zu tun, sondern ist eine generelle Idee, die wie ich finde, nicht mit deinem Satz zusammen gefasst werden kann.

`erst mal nur den kostenlosen Zugang zu Dienstleistungen wie Facebook` und das würde Internet.org auch über sich selbst nie so sagen.

Für diese Herangehensweise wurden zwei Begründungen gegeben:\\

> Trotz der Senkung der Internet Kosten kann nicht jegliche Dienstleistung kostenlos bereitgestellt werden, da die dahinterstehende Anbieter Profit erzielen müssen. 

Reviewer 2: `Trotz der Senkung der Internet Kosten` klingt kopiert und englisch.

> Aber durch das kostenlose Bereitstellen grundlegender Dienstleistungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Zugang zum Internet bieten kann).\\

Reviewer 2: ```suggestion
Aber durch das kostenlose Bereitstellen grundlegender Dienstleistungen könnte man gewährleisten, dass die meisten Menschen Zugang zum Internet haben und die Industrie trotzdem gleichzeitig den größten wirtschaftlichen Profit erzielt (wodurch diese wiederum mehr Zugang zum Internet bieten können).\\
```

> Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.(\cite{HumanRight})\\

Reviewer 2: ```suggestion
Obwohl Zuckerberg sagt, dass er keine spezifische Gruppe von grundlegenden Internet Dienstleistungen vorgibt, beschreibt er welche Anwendungen er als grundlegend ansieht.
```

> Seiner Meinung nach sind es die Dienstleistungen, die nicht Daten intensiv sind (hauptsächlich Text basierende Dienstleistungen und einfach gehaltene Apps). 

Reviewer 2: `Daten intensiv` Englisch

> Seiner Meinung nach sind es die Dienstleistungen, die nicht Daten intensiv sind (hauptsächlich Text basierende Dienstleistungen und einfach gehaltene Apps). 

Reviewer 2: ```suggestion
Seiner Meinung nach sind es die Dienstleistungen, die nicht Daten intensiv sind (text basierte oder einfach gehaltene Anwendungen). 
```

> Außerdem sollte man durch die Anwendungen anderen Inhalt entdecken und sinnvoll mehr Daten benutzen können.

Reviewer 2: `sinnvoll mehr Daten benutzen können` so reden Menschen nicht :rocket:

> Außerdem sollte man durch die Anwendungen anderen Inhalt entdecken und sinnvoll mehr Daten benutzen können.

Reviewer 2: ```suggestion
Man sollte durch die Anwendungen anderen Inhalt entdecken und sinnvoll mehr Daten benutzen können.
```

> Dienstleistungen die unter diesen Kriterien fallen sind zum Beispiel Messengers, soziale Netzwerke, Suchmaschinen und Wikipedia.\\

Reviewer 2: ```suggestion
Für Ihn fallen unter diese Kriterien zum Beispiel Messenger, soziale Netzwerke, Suchmaschinen und Wikipedia (Vgl. \cite{HumanRight}).
```

> Also ist Facebook nach Zuckerbergs Meinung eine grundlegende Internet Dienstleistung die vorrangig zu anderen Anwendungen kostenlos zur Verfügung gestellt werden sollte.\\

Reviewer 2: ```suggestion
Also ist Facebook nach Zuckerbergs Meinung eine grundlegende Internet Dienstleistung die vorrangig zu anderen Anwendungen kostenlos zur Verfügung gestellt werden sollte.
```

> Ein weiterer Grund den Zuckerberg für den partiellen kostenlossen Zugang erwähnt hat ist:\\

Reviewer 2: ```suggestion
Auch findet er:\\
```

> "many people who have never experienced the internet don’t know what a data plan is or why they’d want one. 

Reviewer 2: ```suggestion
``many people who have never experienced the internet don’t know what a data plan is or why they’d want one. 
```

However, most people have heard of services like Facebook and messaging and they want access to them. If we can provide people 
with access to these services, then they’ll discover other content they want and begin to use 

> and understand the broader internet." (vgl. \cite{HumanRight})

Reviewer 2: ```suggestion
and understand the broader internet." \parencite{HumanRight}
```

\medskip

> Facebooks Plan ist es, unter dem Projekt Internet.org, diese Punkte Industrie übergreifend zu bearbeiten. \\

Reviewer 2: ```suggestion
Facebooks Plan ist es, unter dem Projekt Internet.org, diese Punkte Industrie übergreifend zu bearbeiten.
```

Auf der offizielen Facebook Seite für Internet.org wurde das Projekt wie folgt vorgestellt:\\

> 

Reviewer 2: ```suggestion
```

> ``Internet.org is a global partnership between technology leaders, nonprofits, local communities and experts who are working together to bring the internet to the two thirds of the world's population that doesn't have it. Sharing tools, resources and best practices, Internet.org partners will explore solutions in three major opportunity areas: affordability, efficiency and business models.\#ConnectTheWorld" (\cite{InternetOrg})\\

Reviewer 2: ```suggestion
``Internet.org is a global partnership between technology leaders, nonprofits, local communities and experts who are working together to bring the internet to the two thirds of the world's population that doesn't have it. Sharing tools, resources and best practices, Internet.org partners will explore solutions in three major opportunity areas: affordability, efficiency and business models.\#ConnectTheWorld" \parencite{InternetOrg}\\
```

Reviewer 2: Wäre ein guter Abschluss für den Abschnitt.

Außerdem passt der nächst Paragraph inhaltlich nicht wirklich, den kann man viel besser Weiter oben integrieren (grundung am Anfang irgendwo, Facebook Zero, Facebook für jedes Handy wenn über den 1. oder 2. Punkt gesprochen wird.)
